src/neo/Network/P2P/TaskManager.cs

```diff
@@ -94,9 +94,6 @@ private void OnNewTasks(InvPayload payload)
         {
             if (!sessions.TryGetValue(Sender, out TaskSession session))
                 return;
-            // Do not accept payload of type InventoryType.TX if not synced on best known HeaderHeight
-            if (payload.Type == InventoryType.TX && Blockchain.Singleton.Height < sessions.Values.Max(p => p.LastBlockIndex))
-                return;
```

Comment:
The reason for this line is that when a node falls below the best block height, it's better to synchronize the blocks first.